### PR TITLE
feat: cel macro documentation

### PR DESCRIPTION
**Pull request by adityathebe**
resolves: https://github.com/flanksource/docs/issues/148

### CODE REVIEW

File: canary-checker/docs/scripting/cel.md [hunk 1]
```diff
@@ -607,7 +609,7 @@ time.ZoneOffset(); // Could evaluate to -480 for PST
 Syntax:
 
 ```javascript
-time.Parse(layout, value)
+time.Parse(layout, value);
```
Comment (moshloop) [inline canary-checker/docs/scripting/cel.md:612]:
```suggestion
time.Parse(layout, value)
```

File: canary-checker/docs/scripting/cel.md [hunk 2]
```diff
@@ -633,8 +635,9 @@ time.Parse("15:04 02-01-2006", "14:30 26-09-2023"); // Includes time of day info
 `time.ParseLocal` parses a given string into a time object according to a specified layout and the local time zone. It's useful for working with local times.
 
 ```javascript
-time.ParseLocal(layout, value)
+time.ParseLocal(layout, value);
```
Comment (moshloop) [inline canary-checker/docs/scripting/cel.md:638]:
```suggestion
time.ParseLocal(layout, value)
```

File: canary-checker/docs/scripting/cel.md [hunk 3]
```diff
@@ -1372,7 +1365,7 @@ regexp.QuoteMeta("[a-z].*"); // "\\[a\\-z\\]\\.\\*"
 Syntax:
 
 ```javascript
-regexp.Replace(pattern, replacement, input)
+regexp.Replace(pattern, replacement, input);
```
Comment (moshloop) [inline canary-checker/docs/scripting/cel.md:1368]:
```suggestion
regexp.Replace(pattern, replacement, input)
```

Comment (adityathebe) [reply canary-checker/docs/scripting/cel.md]: fixed this in all places. It was prettier automatically adding it

**Pull request merged**